packages/dashboard/theme/lumo/vaadin-dashboard-widget-styles.js

```diff
@@ -4,130 +4,115 @@ import '@vaadin/vaadin-lumo-styles/spacing.js';
 import '@vaadin/vaadin-lumo-styles/style.js';
 import '@vaadin/vaadin-lumo-styles/typography.js';
 import '@vaadin/vaadin-lumo-styles/font-icons.js';
+import { addGlobalThemeStyles } from '@vaadin/vaadin-themable-mixin/register-styles';
 import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+/* Global styles */
+const dashboardWidgetProps = css`
+  html {
+    --vaadin-dashboard-widget-background: var(--lumo-base-color);
+    --vaadin-dashboard-widget-border-radius: var(--lumo-border-radius-l);
+    --vaadin-dashboard-widget-border-width: 1px;
+    --vaadin-dashboard-widget-border-color: var(--lumo-contrast-20pct);
+    --vaadin-dashboard-widget-shadow: 0 0 0 0 transparent;
+    --vaadin-dashboard-widget-editable-shadow: var(--lumo-box-shadow-s);
+    --vaadin-dashboard-widget-selected-shadow: 0 2px 4px -1px var(--lumo-primary-color-10pct),
+      0 3px 12px -1px var(--lumo-primary-color-50pct);
+    --vaadin-dashboard-drop-target-background-color: var(--lumo-primary-color-10pct);
+    --vaadin-dashboard-drop-target-border: 1px dashed var(--lumo-primary-color-50pct);
+  }
+`;
+addGlobalThemeStyles('dashboard-widget-props', dashboardWidgetProps);
+
+/* Styles shared between widgets and sections */
 const dashboardWidgetAndSection = css`
+  /* stylelint-disable rule-empty-line-before */
+  /* stylelint-disable length-zero-no-unit */
+
   :host {
-    border-radius: var(--lumo-border-radius-l);
     color: var(--lumo-body-text-color);
     font-family: var(--lumo-font-family);
     font-size: var(--lumo-font-size-m);
     line-height: var(--lumo-line-height-m);
     --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
     --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
-    /* default max value for the focus ring spacing offset. calc doesn't support unitless 0. */
-    /* stylelint-disable length-zero-no-unit */
-    --_focus-ring-spacing-max-offset: 0px;
-    /* Calculates the offset by which the focus ring should be shifted inwards based on a custom --vaadin-dashboard-gap or --vaadin-dashboard-padding values.
-    Effectively keeps the focus ring visible if --vaadin-dashboard-gap or --vaadin-dashboard-padding is set to 0px */
-    --_focus-ring-spacing-offset: min(
```

Comment:
I think removing this causes the section focus outline to get clipped when padding is set to 0px

![Screenshot 2024-11-28 at 9 49 42](https://github.com/user-attachments/assets/fd4a8837-c5b6-4b30-8ebf-cc4584d928fb)

Reply:
Indeed, I've now refactored the section focus ring to a pseudo with appropriate offsetting, so that it's sufficiently pulled in and also overlays the widgets in the section.